repose-aggregator/components/services/phone-home-service/src/main/scala/org/openrepose/core/services/phonehome/PhoneHomeService.scala

```diff
@@ -154,14 +156,19 @@ class PhoneHomeService @Inject()(@Value(ReposeSpringProperties.CORE.REPOSE_VERSI
         )
       ))
 
-      msgLogger.info(updateMessage)
+      MsgLogger.info(updateMessage)
 
       updateMessage
     }
 
     def sendUpdateMessage(collectionUri: String, message: String): Unit = {
       logger.trace("sendUpdateMessage method called")
 
+      val scope = tracer.buildSpan(TracingOperationName)
+        .withTag(Tags.HTTP_URL.getKey, collectionUri)
+        .ignoreActiveSpan()
+        .startActive(true)
```

Comment:
I will note that the much (but not all) of the work captured by this span will also be captured by the HttpClient Interceptor span. This span will always be a root span. While ideally this span would also be a parent of the HttpClient span, I do not believe that is guaranteed since the `AkkaServiceClient` handles requests in `Future`s, which are likely to be computed on a different thread making it impossible to access the active span through the `ThreadLocalScopeManager` (although a reference to the parent span can be passed directly, if desired).

I am actually a bit worried about the relationship between spans created in the HttpClient Interceptor and spans created outside of it. Because `ThreadLocal` and `Future`s (really, the `ExecutionContext` of `Future`s). I wonder if we want to make those spans ignore the active span and optionally set a parent span that is passed to the client? Of course, that would require further modification of our take on Jaeger's `TracingRequestInterceptor`.

If we did do such a thing, I kind of question the value of the "phone-home" span. The boundary crossing will be captured by the HttpClient Interceptor span anyway. I guess the operation name might be clearer here than in the Interceptor since it has to be more generic, but even so.